common/actions/config/actionCreators.ts

```diff
@@ -69,6 +69,16 @@ export function changeNodeForce(payload: string): interfaces.ChangeNodeForceActi
   };
 }
 
+export type TChangeNetworkIntent = typeof changeNetworkIntent;
```

Comment:
I'd really rather stick to the `REQUESTED` `SUCCEEDED` `FAILED` convention here. When debugging node changing in the past, it does get confusing when you see `NODE_CHANGE` but that was actually just for the reducers, and `NODE_CHANGE_INTENT` is what the sagas acted on. Just seems more obvious with the latter 3 postfixes.

Reply:
Agreed, I'll bring this and `nodeChangeIntent` inline with that.

Reply:
Addressed in https://github.com/MyCryptoHQ/MyCrypto/pull/1652